components/atoms/ActionButton.test.js

```diff
@@ -28,3 +34,9 @@ it("renders ActionButton disabled", () => {
   expect(screen.getByRole("button")).toHaveTextContent(Disabled.args.text);
   expect(screen.getByRole("button")).toHaveAttribute("disabled");
 });
+
+it("renders ActionButton disabled", () => {
```

Comment:
Unit test title should be descriptive to what it does. (copy paste error here)

Optional: Can you wrap the unit tests in a describe() clause? Look at DrawerItem.test.js for a super-simple example.